src/rules/collection-method-value.js

```diff
@@ -42,4 +42,4 @@ module.exports = function (context) {
             }
         }
     }
-}
```

Comment:
whoops, I can undo this if you want. I edited the file to remove the semi-colons to lint would pass, before you pushed the fixup commits...